fast/stages/0-bootstrap/cicd.tf

```diff
@@ -86,14 +86,14 @@ module "automation-tf-cicd-sa" {
     "roles/iam.workloadIdentityUser" = [
       each.value.branch == null
       ? format(
-        local.workload_identity_providers_defs[each.value.type].principal_repo,
+        local.workload_identity_providers_defs[each.value.type].principal_subject,
```

Comment:
one step forward, but I would still iterate on naming a bit; copy/pasting from chat: "I think we need one more naming iteration, as subject and branch seem two different abstractions; we had repo and branch which both refer to the actual SCM used, now we have subject and branch which refer one to an attribute in the token, the other to the SCM; let's settle on one or the other"